extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -295,6 +335,13 @@ export class AmpLightboxViewer extends AMP.BaseElement {
   setupElement_(element) {
     this.updateStackingContext_(element, /* reset */ false);
     element.classList.add('amp-lightboxed');
+    // update description box
+    const descText = this.manager_.getDescription(element);
```

Comment:
yeah just remove the `hide` class on close. I think description should be visible on every open action. might change after UX review though